custom/icds_reports/migrations/0189_new_fields_to_bihar_demogrpahics.py

```diff
@@ -45,5 +45,3 @@ class Migration(migrations.Migration):
             field=models.SmallIntegerField(null=True),
         ),
     ]
-
-    operations.extend(get_view_migrations())
```

Comment:
did you intend to commit this?

Reply:
Yes, I moved `get_view_migrations()` to the latest migration file. Isn't that an expected change?

Reply:
Oh that makes sense. I'm not familiar with the process